src/sql/workbench/contrib/resourceViewer/browser/media/resourceViewerView.css

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.resource-view {
```

Comment:
this name is a little generic (i.e. I could imagine someone adding a conflicting ID in the future).  Maybe something like `resource-explorer-view` would be better scoped?